### ISSUE TITLE
Error loading materials using FixedPipeline LightMode in jme 3.2

### ISSUE THREAD

**Issue by Ali-RS** [Simsilica/SimArboreal-Editor#3]
Hi 
FixedPipeline light mode is removed in jme 3.2 so running SimArboreal-Editor with jme 3.2 jars I am getting 

```
java.lang.UnsupportedOperationException
	at com.jme3.material.plugins.J3MLoader.readTechnique(J3MLoader.java:655)
	at com.jme3.material.plugins.J3MLoader.loadFromRoot(J3MLoader.java:777)
	at com.jme3.material.plugins.J3MLoader.load(J3MLoader.java:798)
	at com.jme3.asset.DesktopAssetManager.loadLocatedAsset(DesktopAssetManager.java:259)
	at com.jme3.asset.DesktopAssetManager.loadAsset(DesktopAssetManager.java:373)
	at com.jme3.material.Material.<init>(Material.java:108)
	at com.simsilica.arboreal.GroundState.initialize(GroundState.java:130)
	at com.simsilica.lemur.event.BaseAppState.initialize(BaseAppState.java:75)
	at com.jme3.app.state.AppStateManager.initializePending(AppStateManager.java:251)
	at com.jme3.app.state.AppStateManager.update(AppStateManager.java:281)
	at com.jme3.app.SimpleApplication.update(SimpleApplication.java:236)
	at com.jme3.system.lwjgl.LwjglAbstractDisplay.runLoop(LwjglAbstractDisplay.java:151)
	at com.jme3.system.lwjgl.LwjglDisplay.runLoop(LwjglDisplay.java:193)
	at com.jme3.system.lwjgl.LwjglAbstractDisplay.run(LwjglAbstractDisplay.java:232)
	at java.lang.Thread.run(Thread.java:745)

```

should I make a PR and change all 

```
Technique {
        LightMode FixedPipeline
    }


```

to SinglePass, in following materials ? 

- MultiResolution.j3md (in SimArboreal-Editor)
- AxisBillboardLighting.j3md (in SimArboreal)
- LeafLighting.j3md (in SimArboreal)
- TreeLighting.j3md (in SimArboreal)
- IndexedBillboardLighting.j3md (in SimArboreal)

**Comment by pspeed42** [comment]
I think those techniques can just be removed, right?  There is already a multipass.  

I mean, adding single pass techniques might be nice but that's not the same as the FixedPipeline.  There is no fixed pipeline anymore so techniques specific to that should just go away I guess.

**Comment by Ali-RS** [comment]
Hm, yes, I removed them and it works fine now. 
Should I make a PR ?

**Comment by pspeed42** [comment]
That would be great.

**Comment by Ali-RS** [comment]
Fixed #4
